svgo.config.js

```diff
@@ -24,35 +42,11 @@ module.exports = {
         ]
       }
     },
-    {
-      name: 'convertColors',
-      params: {
-        currentColor: 'red'
-      }
-    },
-    {
-      name: 'inlineStyles',
-      params: {
-        onlyMatchedOnce: false
-      }
-    },
     {
       name: 'removeAttrs',
       params: {
         attrs: '(baseProfile|class|clip-rule|id|stroke-miterlimit|version)'
       }
-    },
-    { name: 'removeTitle' },
-    {
-      name: 'removeViewBox',
-      active: false
-    },
-    {
-      name: 'removeUnknownsAndDefaults',
-      params: {
-        unknownAttrs: false
-      }
-    },
-    { name: 'sortAttrs' }
```

Comment:
With SVGO v3, some of the built-in plugin configuration has been moved to `preset-default` plugin. For `sortAttrs` and `removeTitle` those are now enabled by default and no longer need to be explicitly added.